### ISSUE TITLE
Admob 7.1.0 - iOS - ITMS-90427: Invalid Swift Support - The expected dylibs are missing

### ISSUE THREAD

**Issue by dcosmin-97** [googleads/googleads-mobile-unity#2126]
Environment:

    Unity version: 2020.3.36f1
    Google Mobile Ads Unity plugin version: 7.1.0
    Platform: iOS
    Platform OS version: 15+
    Mediation ad networks used, and their versions:
    -> AdColony 2.6.0
    -> AppLovin 6.3.0
    -> Facebook 3.8.0
    -> Unity 3.2.1
    Other SDK : Firebase 9.1.0
    External Dependency Manager: 1.2.171

Problem:
After solving the problem with The bundle at ‘App/Frameworks/UnityFramework.framework' contains disallowed file 'Frameworks',
I am receiving this error from AppStore Connect and I cannot publish an update: ITMS-90427: Invalid Swift Support - The expected dylibs are missing from the app’s Framework location, such as /Payload/App.app/Frameworks. 

I have tried multiple things and nothing is working. 

Has anyone faced this issue and found a solution?
Thank you!

**Comment by Claptrap725** [comment]
I am having the EXACT same issue. I have tried so many different things. Have you found a solution yet? @dcosmin-97

**Comment by Claptrap725** [comment]
I found a solution here [https://forum.unity.com/threads/2019-3-validation-on-upload-to-store-gives-unityframework-framework-contains-disallowed-file.751112/](https://forum.unity.com/threads/2019-3-validation-on-upload-to-store-gives-unityframework-framework-contains-disallowed-file.751112/)  specifically from reply [#38]() from a user called [Melnikovv](https://forum.unity.com/members/melnikovv.1185096/)

_In XCode_
Set **yes** on Always Embed Swift Standard Libraries in target Unity-iPhone in Build Settings.
Set **no** on Always Embed Swift Standard Libraries in target UnityFramework in Build Settings.

This worked for me. 

I also had another issue with Unity In App Purchasing that required me to downgrade to version 3.1.0. I think that is an unrelated issue but I'm putting it here just in case. 
Hope this helps someone

**Comment by AftabUfaq** [comment]
> I found a solution here https://forum.unity.com/threads/2019-3-validation-on-upload-to-store-gives-unityframework-framework-contains-disallowed-file.751112/ specifically from reply #38 from a user called [Melnikovv](https://forum.unity.com/members/melnikovv.1185096/)
> 
> _In XCode_ Set **yes** on Always Embed Swift Standard Libraries in target Unity-iPhone in Build Settings. Set **no** on Always Embed Swift Standard Libraries in target UnityFramework in Build Settings.
> 
> This worked for me.
> 
> I also had another issue with Unity In App Purchasing that required me to downgrade to version 3.1.0. I think that is an unrelated issue but I'm putting it here just in case. Hope this helps someone

this solve all the issue and my app got approved by apple store. but there is some issue on some devices that app crash even before starting. there is no crash report in Xcode, or App Store connect. don't know how to solve this now. :(

**Comment by FocusZheng** [comment]
@AftabUfaq  i got the problem same as you, it can work in debug mode but crash in release mode.did you solve this problem?
